features/gezag-minderjarige/1.3a-is-geboren-in-buitenland.feature

```diff
@@ -0,0 +1,103 @@
+      * is meerderjarig
+      En de persoon 'Paul' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Regina' en 'Paul' zijn met elkaar gehuwd
+      En de persoon 'Nona' met burgerservicenummer '000000036'
+      * is minderjarig
+      * heeft 'Regina' als ouder 1
+      * heeft 'Paul' als ouder 2
+
+  Regel: Kind is geboren in het buitenland
+
+    Scenario: Het kind is geboren in China er is sprake van GezagNietTeBepalen
+      Gegeven persoon 'Nona' 
+      * is geboren in het buitenland
+      * verblijft in Nederland
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                                                                                                                                             |
+      | type        | GezagNietTeBepalen                                                                                                                                                                                                 |
+      | toelichting | gezag is niet te bepalen omdat minderjarige in het buitenland is geboren en geen Nederlandse adoptie-akte bekend is. |
+
+    Scenario: Het kind is geboren in China en verblijft niet in Nederland er is sprake van GezagNietTeBepalen
+              De toelichting komt niet overeen met de verwachting, de verblijfplaats zou niet nodig zijn bij de bepaling
+              Hier voor is issue: https://github.com/BRP-API/brp-api-gezag/issues/204 aangemaakt
+      Gegeven persoon 'Nona' 
+      * is geboren in het buitenland
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                                                                                                                                             |
+      | type        | GezagNietTeBepalen                                                                                                                                                                                                 |
+      | toelichting | gezag is niet te bepalen omdat de volgende relevante gegevens ontbreken: verblijfplaats van bevraagde persoon |
+
+  Regel: Kind is niet geboren in het buitenland
```

Comment:
Taal: ontkenning zoveel mogelijk vermijden.

```suggestion
  Regel: Kind is geboren in het binnenland
```
of
```suggestion
  Regel: Kind is geboren in Nederland
```

Reply:
ik stem voor "geboren in Nederland". Den Haag ligt aan de kust, niet in het "binnenland"